flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/SpillableSubpartition.java

```diff
@@ -246,9 +246,9 @@ public int unsynchronizedGetNumberOfQueuedBuffers() {
 	@Override
 	public String toString() {
 		return String.format("SpillableSubpartition [%d number of buffers (%d bytes)," +
-						"finished? %s, read view? %s, spilled? %s]",
-				getTotalNumberOfBuffers(), getTotalNumberOfBytes(), isFinished, readView != null,
-				spillWriter != null);
+				"%d backlog, finished? %s, read view? %s, spilled? %s]",
```

Comment:
`"%d buffers in backlog, finished (...)"` ? 
`"backlog = %d, finished (...)"` ? 
`"%d in backlog, finished (...)"` ? 

`%d backlog` is a little bit cryptic.